### PR TITLE
Update ru.yml

### PR DESCRIPTION
**Pull request by Spiralman666**
I was working on it 2 hours to be sure that everything looks and reads as credible as it can be! Hope you like it cause original google-translated version is awfull!

### CODE REVIEW

File: _data/langs/ru.yml [hunk 1]
```diff
@@ -6,41 +6,41 @@ head:
...
   about:
-    header: "Зачем останавливаться?"
-    paragraph: "Verge - это децентрализованная валюта, предназначенная для пользователей, которые ценят конфиденциальность. Он улучшает исходную блокировку биткойнов с помощью нескольких сетей с анонимностью, таких как Tor и i2p. Он использует поддержку нескольких алгоритмов для повышения безопасности и обеспечения равного распределения монет шахтерам."
-  resources:
+    header: "Почему Verge?"
+    paragraph: "Verge - это децентрализованная валюта, предназначенная для пользователей, которые ценят конфиденциальность. Она усовершенствовала оригинальный блокчейн биткоина, за счёт использования анонимных сетей, таких как Tor и i2p. Она поддерживает несколько алгоритмов майнинга для повышения безопасности и обеспечения равного распределения вознаграждения между майнерами."
+  resources: 
     header: "Ресурсы"
-    black_paper: "Черная бумага"
+    black_paper: ""Черная" книга"
```
Comment (gfranko) [inline _data/langs/ru.yml]: Could you change this to: `"\"Черная\" книга"`

File: _data/langs/ru.yml [hunk 2]
```diff
@@ -50,87 +50,87 @@ exchanges:
...
+  electrum_windows_tor: "TOR Windows Electrum Verge-Кошелёк"
+  paper: "Бумажный Verge-Кошелёк"
+  daemon_docker: "Образ демона Докера для Verge"
   coinomi: "Coinomi Мультивалютный Android-кошелек"
 roadmap:
   header: "Дорожная карта"
   blackpaper:
-    header: "Черная бумага"
-    date: "<font color='red'>ВЫПУЩЕННЫЙ</font> 4 июня 2017 года"
+    header: ""Черная" книга"
```
Comment (gfranko) [inline _data/langs/ru.yml]: Could you change this to: `"\"Черная\" книга"`

File: _data/langs/ru.yml [hunk 3]
```diff
@@ -50,87 +50,87 @@ exchanges:
...
-    header: "Electrum TOR Server/Client"
-    date: "<font color='red'>ВЫПУЩЕННЫЙ</font> 5 июня 2017 года"
+    header: "Electrum TOR клиент-сервер"
+    date: "<font color='red'>ВЫПУЩЕННО</font> 5 июня 2017 года"
   electrum_binaries:
-    header: "Electrum Binaries"
-    paragraph: "Binary clients for easy-to-use lite wallets."
-    date: "<font color='red'>ВЫПУЩЕННЫЙ</font> 7 июня 2017 года"
+    header: "Electrum бинарные ресурсы"
+    paragraph: "Двоичные клиенты для использования с "облегчёнными" кошельками."
```
Comment (gfranko) [inline _data/langs/ru.yml]: Could you change this to: `"Двоичные клиенты для использования с \"облегчёнными\" кошельками."`

File: _data/langs/ru.yml (hunk 2 above)
Comment (Spiralman666) [inline _data/langs/ru.yml]: can you explain me what do i need to do?

Comment (gfranko) [inline _data/langs/ru.yml]: You basically just need to put in a backslash before the nested double quotes (this properly escapes them)

Comment (Spiralman666) [inline _data/langs/ru.yml]: aha i see that "" are used as syntax so to separate syntax from text we use here // right?

Comment (gfranko) [inline _data/langs/ru.yml]: We put `\` in front of a nested double quote. The other option is to use single quotes.

Comment (gfranko) [inline _data/langs/ru.yml]: If you just update those strings with my suggestions, the build should pass

File: _data/langs/ru.yml [hunk 4]
```diff
@@ -25,7 +25,7 @@ info:
     paragraph: "Verge - это децентрализованная валюта, предназначенная для пользователей, которые ценят конфиденциальность. Она усовершенствовала оригинальный блокчейн биткоина, за счёт использования анонимных сетей, таких как Tor и i2p. Она поддерживает несколько алгоритмов майнинга для повышения безопасности и обеспечения равного распределения вознаграждения между майнерами."
   resources: 
     header: "Ресурсы"
-    black_paper: ""Черная" книга"
+    black_paper: "\"Черная"\ книга"
```
Comment (gfranko) [inline _data/langs/ru.yml]: This needs to be: `black_paper: "\"Черная\" книга"`

File: _data/langs/ru.yml [hunk 5]
```diff
@@ -76,7 +76,7 @@ wallets:
 roadmap:
   header: "Дорожная карта"
   blackpaper:
-    header: ""Черная" книга"
+    header: "\"Черная"\ книга"
```
Comment (gfranko) [inline _data/langs/ru.yml]: This needs to be `header: "\"Черная\" книга"`

File: _data/langs/ru.yml [hunk 6]
```diff
@@ -86,7 +86,7 @@ roadmap:
     date: "<font color='red'>ВЫПУЩЕННО</font> 5 июня 2017 года"
   electrum_binaries:
     header: "Electrum бинарные ресурсы"
-    paragraph: "Двоичные клиенты для использования с "облегчёнными" кошельками."
+    paragraph: "Двоичные клиенты для использования с \"облегчёнными"\ кошельками."
```
Comment (gfranko) [inline _data/langs/ru.yml]: This needs to be: `paragraph: "Двоичные клиенты для использования с \"облегчёнными\" кошельками."`

**Pull request merged**